sssom/util.py

```diff
@@ -743,7 +716,7 @@ def read_pandas(file: Union[str, TextIO], sep: Optional[str] = None) -> pd.DataF
         else:
             sep = "\t"
             logging.warning("Cannot automatically determine table format, trying tsv.")
-    return read_csv(file, comment="#", sep=sep).fillna("")
+    return pd.read_csv(file, comment="#", sep=sep).fillna("")
```

Comment:
The reason why `read_csv` was created was due to pandas omitting rows within the `DataFrame` of a `MappingSetDataFrame` object which would be part of the data and not a comment. @matentzn and I had discussed this while implementing the function.

Reply:
was that because the data had the comment character (`#`) in it?

Reply:
Yes.

Reply:
Yes this looks like a very dangerous change

Reply:
okay then i'll revert it.

Reply:
Reverted in cf80b47